apps/passport-server/migrations/33_events_to_tg_chat_many_to_one.sql

```diff
@@ -0,0 +1,5 @@
+-- First, drop the existing unique index on ticket_event_id and telegram_chat_id
+ALTER TABLE telegram_bot_events DROP CONSTRAINT telegram_bot_events_ticket_event_id_telegram_chat_id_key;
+
+-- Next, add a new unique index on ticket_event_id
+ALTER TABLE telegram_bot_events ADD UNIQUE (ticket_event_id)
```

Comment:
note that this will cause staging, prod, or local environments to fail a migration if they have a many:many relationship already. we need to think of a plan for handling this